moved keys to testdata.json

rrrooommmaaa opened the pull request:
This PR simplifies structure of test-secrets,
some non-secret data is moved to testdata.json

close #3246

rrrooommmaaa commented:
@tomholub Looks like enterprise-mock is using this URL for google auth:
'chrome-enterprise': 'https://google.mock.flowcryptlocal.com:8001'
Can you please explain the idea behind this?

tomholub commented:
The reason for that hostname is that enterprise extension has reduced scope and permissions (compare manifest.json files in built folders between consumer and enterprise). These reduced permissions cause a different treatment of CORS or other security policies, which in turn prevent calling to a localhost API, for whatever reason.

Either that, or maybe certain issues only manifested on enterprise addon when calling a "real" host that is non-localhost. I forgot exactly.

tomholub reviewed (commented):
Code is in the right direction

rrrooommmaaa commented:
Looks like   `ci_dev_account`, `data_encryption_password` and `ldap_refesh_test` are not used in our codebase, so I cleaned up `TestSecretsInterface`, also removed `keyInfo` section.
